docs/user/aws/install_upi.md

```diff
@@ -50,6 +50,21 @@ $ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml openshift/99_o
 
 You are free to leave the compute MachineSets in if you want to create compute machines via the machine API, but if you do you may need to update the various references (`subnet`, etc.) to match your environment.
 
+### Make control-plane nodes unschedulable
+
+Currently [emptying the compute pools](#empty-compute-pools) makes control-plane nodes schedulable.
+But due to a [Kubernetes limitation][kubernetes-service-load-balancers-exclude-masters], router pods running on control-plane nodes will not be reachable by the ingress load balancer.
```

Comment:
routers are now ingress controllers :-)

Reply:
> routers are now ingress controllers :-)

The pod is [still called `router`][1].  But is "ingress controller pod" the preferred English?  Do we have plans to rename the pods to match?

[1]: https://github.com/openshift/cluster-ingress-operator/blob/4e9047dc81cfa359d61db1d9ec78bd2b144e9b5b/pkg/operator/controller/names.go#L35

Reply:
We have "Ingress router pods" in the [official docs][1], so if any rephrasing is needed we'd probably want to bump those too.

[1]: https://docs.openshift.com/container-platform/4.1/installing/installing_bare_metal/installing-bare-metal.html#network-topology-requirements